astropy/wcs/tests/test_utils.py

```diff
@@ -353,6 +370,59 @@ def test_wcs_to_celestial_frame():
     assert isinstance(frame, Galactic)
 
 
+def test_wcs_to_body_frame():
+    # Import astropy.coordinates here to avoid circular imports
+    from astropy.coordinates import BaseCoordinateFrame
+
+    unknown_wcs = WCS(naxis=2)
+    unknown_wcs.wcs.ctype = ["UTLN-TAN", "UTLT-TAN"]
+    with pytest.raises(KeyError, match="unknown solar system object abbreviation UT"):
+        frame = wcs_to_celestial_frame(unknown_wcs)
+
+    triaxial_wcs = WCS(naxis=2)
+    triaxial_wcs.wcs.ctype = ["MELN-TAN", "MELT-TAN"]
+    if not WCSLIB_LT8:
+        triaxial_wcs.wcs.aux.a_radius = 2439700.0
+        triaxial_wcs.wcs.aux.b_radius = 2439900.0
+        triaxial_wcs.wcs.aux.c_radius = 2438800.0
+    with pytest.raises(
```

Comment:
This should fail on `WCSLIB_LT_8` - which I guess we are not currently testing...